corehq/apps/hqwebapp/static/cloudcare/less/formplayer-common/appicon.less

```diff
@@ -22,6 +22,10 @@
       line-height: @icon-height;
       font-size: @icon-height;
     }
+    .appicon-custom {
+      top: 0.05*@icon-height + 10;
```

Comment:
Note: This 10px buffer is there to replicate the fact that the CC flower right now has its own internal / natural padding on top of the layout.

Without this, there's a really awkward situation where users need to add a default padding to the top of their image but not the bottom, which is a very weird thing to force them to do

Reply:
I like that. The one production app I've seen using this has an icon that bumps up against the top of the tile and it doesn't look right.

Reply:
Having said that, can you move this rationale into a comment?

Reply:
Good call. done